server.js

```diff
@@ -1,26 +1,47 @@
-const express = require('express')
-const app = express()
-const mongoose = require('mongoose')
-const port = 3000
+const express = require('express');
+const mongoose = require('mongoose');
+const app = express();
+const port = 3000;
+const route = require('./routes');
+const arr = require("./data.json");
+const { post, get, del } = require('./routes');
+
+app.use(express.json());
 
 const uri = 'mongodb+srv://Surya:sun@asapcluster.3osvgms.mongodb.net/?retryWrites=true&w=majority';
 
+// Connect to MongoDB
+mongoose.connect(uri)
+  .then(() => {
+    console.log('Connected to MongoDB');
+  })
+  .catch(error => {
+    console.error('Error connecting to MongoDB:', error);
+  });
 
+// CRUD routes
+app.get('/get', get);
+app.post('/post', post);
+app.delete('/:id', del);
 
-app.get('/' ,(req,res) =>{
-    mongoose.connect(uri)
-    .then(() =>{
-        res.json({Connection:'Connected'})
+app.get('/', (req, res) => {
```

Comment:
The MongoDB connection is being established every time a request is made to the '/' route. It would be more efficient to establish the connection when the server starts. [important]